_data/localization.json

```diff
@@ -8,111 +8,132 @@
+      "sv": "Det är ett fel i källkoden: referensmplementeringen av det reguljära uttrycket kan inte användas. Vänligen klicka på 'förbättra sidan' nedan för att fixa problemet själv, eller klicka på 'rapportera problem' får att skriva en felrapport så att vi kan kolla på problemet. ",
+      "pt-br": "Existe um erro no código fonte: a referência de implementação da expressão regular não pode ser usada. Por favor clique em 'melhore a página' para resolver esse problema você mesmo, ou então clique em 'reporte um problema' para submeter um reporte de erro então nós iremos olhar isto. "
     }
   },
   "license": {
     "en": "<span xmlns:dct='http://purl.org/dc/terms/' href='http://purl.org/dc/dcmitype/InteractiveResource' property='dct:title' rel='dct:type'>Regex Tutorial</span> by <span xmlns:cc='http://creativecommons.org/ns#' property='cc:attributionName'>Nicco Kunzmann</span> is licensed under a <a rel='license' href='http://creativecommons.org/licenses/by/4.0/'>Creative Commons Attribution 4.0 International License</a>.",
     "de": "<span xmlns:dct='http://purl.org/dc/terms/' href='http://purl.org/dc/dcmitype/InteractiveResource' property='dct:title' rel='dct:type'>Regex-Tutorial</span> von <span xmlns:cc='http://creativecommons.org/ns#' property='cc:attributionName'>Nicco Kunzmann</span> ist lizenziert unter einer <a rel='license' href='http://creativecommons.org/licenses/by-sa/4.0/'>Creative Commons Namensnennung - Weitergabe unter gleichen Bedingungen 4.0 International Lizenz</a>.",
-    "sv": "<span xmlns:dct='http://purl.org/dc/terms/' href='http://purl.org/dc/dcmitype/InteractiveResource' property='dct:title' rel='dct:type'>Regex Tutorial</span> av <span xmlns:cc='http://creativecommons.org/ns#' property='cc:attributionName'>Nicco Kunzmann</span> är licenserad under en <a rel='license' href='http://creativecommons.org/licenses/by/4.0/'>Creative Commons Erkännande 4.0 Internationell Licens</a>."
+    "sv": "<span xmlns:dct='http://purl.org/dc/terms/' href='http://purl.org/dc/dcmitype/InteractiveResource' property='dct:title' rel='dct:type'>Regex Tutorial</span> av <span xmlns:cc='http://creativecommons.org/ns#' property='cc:attributionName'>Nicco Kunzmann</span> är licenserad under en <a rel='license' href='http://creativecommons.org/licenses/by/4.0/'>Creative Commons Erkännande 4.0 Internationell Licens</a>.",
+    "pt-br": "<span xmlns:dct='http://purl.org/dc/terms/' href='http://purl.org/dc/dcmitype/InteractiveResource' property='dct:title' rel='dct:type'>Regex Tutorial</span> por <span xmlns:cc='http://creativecommons.org/ns#' property='cc:attributionName'>Nicco Kunzmann</span> está licenciado sob a <a rel='license' href='http://creativecommons.org/licenses/by/4.0/'>Creative Commons Atribuições 4.0 Licença Internacional </a>."
```

Comment:
If you like, you are welcom to add your name to the license of your translation.